doc/Developers-Intro.md

```diff
@@ -42,29 +42,45 @@ If you have seen Friendica you probably have ideas to improve it, haven't you?
 * Make plans for a better Friendica interface design and share them with us.
 * Tell us if you are able to realize your ideas or what kind of help you need.
 We can't promise we have the right skills in the group but we'll try.
-* Choose a thing to start with, e.g. work on the icon set of your favourite theme
+* Choose a thing to start with, e.g. work on the icon set of your favorite theme
 
 Programming
 ---
 
+###Coding standards
+
+For the sake of consistency between contribution and general code readability, Friendica follows the widespread [PSR-2 coding standards](http://www.php-fig.org/psr/psr-2/).
+Don't worry, you don't have to know by heart the PSR-2 coding standards to start contributing to Friendica.
+Before comitting your work, you can use the PHP Coding Standard Fixer located in the `util` folder this way:
+
+	$> php util/php-cs-fixer.phar fix <file or folder> --level=psr2
```

Comment:
For what it's worth, this tool is standalone, it just needs PHP to be executed, which is required for Friendica anyway, but I totally get your point. I'll do some rewriting.